examples/minimal-logs/main.tf

```diff
@@ -0,0 +1,156 @@
+  policy_arn = data.aws_iam_policy.cw-agent-server-policy.arn
+}
+
+# Set up VPC & subnet
+resource "aws_vpc" "tamr_vm_vpc" {
+  cidr_block           = "10.0.0.0/16"
+  enable_dns_support   = true
+  enable_dns_hostnames = true
+  tags                 = var.tags
+}
+
+resource "aws_subnet" "tamr_vm_subnet" {
+  vpc_id            = aws_vpc.tamr_vm_vpc.id
+  cidr_block        = "10.0.0.0/24"
+  availability_zone = local.az
+  tags              = var.tags
+}
+
+# Set up HBase logs bucket
+module "s3-bucket" {
+  source             = "git::git@github.com:Datatamer/terraform-aws-s3.git?ref=1.1.0"
+  bucket_name        = format("%s-tamr-module-test-bucket", var.name-prefix)
+  read_write_actions = local.tamr_vm_s3_actions
+  read_write_paths   = [""] # r/w policy permitting specified rw actions on entire bucket
+  additional_tags    = var.tags
+}
+
+# Create new EC2 key pair
+#resource "tls_private_key" "tamr_ec2_private_key" {
+#  algorithm = "RSA"
+#}
+
+module "aws-vm-sg-ports" {
+  #source = "git::https://github.com/Datatamer/terraform-aws-tamr-vm.git//modules/aws-security-groups?ref=2.0.0"
+  source = "../../modules/aws-security-groups"
+}
+
+module "aws-sg" {
+  source = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  vpc_id = aws_vpc.tamr_vm_vpc.id
+  ingress_cidr_blocks = [
+    "10.0.0.0/24"
+  ]
+  egress_cidr_blocks = [
+    "0.0.0.0/0"
+  ]
+  ingress_ports    = module.aws-vm-sg-ports.ingress_ports
+  sg_name_prefix   = var.name-prefix
+  ingress_protocol = var.ingress_protocol
+  egress_protocol  = "all"
+  tags             = var.tags
+}
+
+module "tamr-vm" {
+  source                      = "git::git@github.com:Datatamer/terraform-aws-tamr-vm.git?ref=4.2.0"
+  aws_role_name               = format("%s-tamr-ec2-role", var.name-prefix)
+  aws_instance_profile_name   = format("%s-tamr-ec2-instance-profile", var.name-prefix)
+  aws_emr_creator_policy_name = format("%sEmrCreatorPolicy", var.name-prefix)
+  s3_policy_arns = [
+    module.s3-bucket.rw_policy_arn,
+  ]
+  ami               = var.ami_id
+  instance_type     = "r5.2xlarge"
+  key_name          = var.key_name
+  availability_zone = local.az
+  vpc_id            = aws_vpc.tamr_vm_vpc.id
+  subnet_id         = aws_subnet.tamr_vm_subnet.id
+  bootstrap_scripts = [
+
+    # NOTE: If you would like to use local scripts, you can use terraform's file() function
+    templatefile("./test-bootstrap-scripts/cloudwatch-install.sh", { region = data.aws_region.current.name, endpoint = module.endpoints.endpoints["logs"].dns_entry[0]["dns_name"] }),
+  ]
+
+  security_group_ids = module.aws-sg.security_group_ids
+  tags               = var.tags
+}
+
+resource "aws_route_table" "example" {
+  vpc_id = aws_vpc.tamr_vm_vpc.id
+  tags   = var.tags
+}
+
+resource "aws_route_table_association" "a" {
+  subnet_id      = aws_subnet.tamr_vm_subnet.id
+  route_table_id = aws_route_table.example.id
+}
+
+module "endpoints" {
```

Comment:
Could we be using the tamr networking module instead? Would that simplify this example?

Reply:
Dan, for the minimal use case there are several attributes that are not required, that is why, in my opinion, creating the VPC with minimal resources is sufficient, for example, `interface_endpoint_ingress_sg`, `compute_subnet_cidr_block`, `data_subnet_cidr_blocks`, `EMR endpoint` all of those are not required....we can use the networking module for the complete example when working on collecting the logs for EMR and Spark. Would you approve of not using the networking module instead?

Reply:
I think it might be a good idea to use the networking module as much as possible since it might give us ideas on new features to add to it. 
Even though I understand your point of not being 'minimal' in terms of resources created, it reduces the complexity of creating and managing the VPC in the future by delegating the details to the vpc module.

Reply:
I completely understand the point and will accommodate it with the networking module. Thanks for your time.